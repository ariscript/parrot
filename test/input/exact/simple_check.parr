check:
    print(1) is print(1).

print(0)
