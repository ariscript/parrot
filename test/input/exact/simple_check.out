1
1
check block: 1/1 tests passed
0
0
